packages/jest-emotion/test/react-enzyme.test.js

```diff
@@ -1,60 +1,88 @@
 import 'test-utils/legacy-env'
 /** @jsx jsx */
+import cases from 'jest-in-case'
 import * as enzyme from 'enzyme'
 import { jsx } from '@emotion/core'
-import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
 import { createSerializer } from 'jest-emotion'
-import { toMatchSnapshot } from 'jest-snapshot'
 import React from 'react'
-
-const createEnzymeSnapshotMatcher = serializerOptions => {
-  const serializer = createEnzymeSerializer(serializerOptions)
-  const identityPrinter = v => v
-
-  return function(val) {
-    return toMatchSnapshot.call(this, serializer.print(val, identityPrinter))
-  }
-}
+import toJson from 'enzyme-to-json'
 
 expect.addSnapshotSerializer(createSerializer())
 
-expect.extend({
-  toMatchShallowSnapshot: createEnzymeSnapshotMatcher(),
-  toMatchDeepSnapshot: createEnzymeSnapshotMatcher({ mode: 'deep' })
-})
+const testWithMethods = (description, testCase) =>
```

Comment:
would prefer gathering all those test cases in a single object or something to call `jest-in-case` only once (not that it's heavy or this here is bad, I'd only want to have this written in a similar fashion as other, existing, tests are written already). Look for an example here - https://github.com/emotion-js/emotion/blob/8767a69251c8de86e8350ac8788b4129847e15bb/packages/core/__tests__/keyframes.js#L11